code/ui/manager/src/components/sidebar/Tree.tsx

```diff
@@ -23,6 +30,19 @@ import type { Highlight, Item } from './types';
 
 import { isStoryHoistable, createId, getAncestorIds, getDescendantIds, getLink } from './utils';
 
+type StoryStateKey = State['status'][keyof State['status']][0]['status'];
+const order: StoryStateKey[] = ['unknown', 'pending', 'success', 'warn', 'error'];
```

Comment:
This logic about which statuses are more important should be somewhere else, not deep inside a UI component. 

I think it should be probably be implemented as a function `(State['status'][]) => State['status']` (or `StoryStateKey[] => StoryStateKey`?), rather than assume it'll just be it's a simple order of importance also.